### PR TITLE
Use Swift PM instead of CocoaPods

### PR DESCRIPTION
**Pull request by guillaumerose**
Swift PM is new way of managing dependencies with Swift code. It is integrated with Xcode directly.

### CODE REVIEW

**Comment by anjannath** [comment]
Maybe the version of xcode in travis is older? it seems to pass in circleci

File: build.sh [hunk 1]
```diff
@@ -2,17 +2,13 @@
...
+PROJECT=$BASE_DIR/CodeReady\ Containers.xcodeproj
 SCHEME=CodeReady\ Containers
 
 if [ ! -f "$BUILD_DIR" ]; then
 	mkdir -p "$BUILD_DIR"
 fi
 
 # Build xcarchive
-xcodebuild -workspace "$WORKSPACE" -config Release -scheme "$SCHEME" -archivePath "$ARCHIVE_PATH" archive
+xcodebuild -project "$PROJECT" -config Release -scheme "$SCHEME" -derivedDataPath "$BUILD_DIR"
```
Comment (anjannath) [inline build.sh:13]: 👍

**Comment by guillaumerose** [comment]
The tarball format is not the same, there are more subdirectories now. Is it an issue?

**Comment by anjannath** [comment]
> The tarball format is not the same, there are more subdirectories now. Is it an issue?

No issues!

**Review by anjannath — APPROVED** [review]

**Pull request merged**